gdx-jnigen-gradle/src/main/java/com/badlogic/gdx/jnigen/gradle/JnigenExtension.java

```diff
@@ -82,36 +86,52 @@ public void all(Action<BuildTarget> container) {
 	public void robovm(Action<RoboVMXml> action) {
 		action.execute(robovm);
 	}
-	
-	public void add(TargetOs type) {
-		add(type, false, false, null);
+
+	public void add(Os type) {
+		add(type, Architecture.Bitness._32);
+	}
+
+	public void add(Os type, Architecture.Bitness bitness) {
+		add(type, bitness, Architecture.x86);
 	}
 
-	public void add(TargetOs type, boolean is64Bit) {
+	public void add(Os type, Architecture.Bitness bitness, Architecture architecture) {
+		add(type, bitness, architecture, null);
+	}
+
+	@Deprecated
+	public void add(Os type, boolean is64Bit) {
 		add(type, is64Bit, false, null);
 	}
 
-	public void add(TargetOs type, boolean is64Bit, boolean isARM) {
+	@Deprecated
+	public void add(Os type, boolean is64Bit, boolean isARM) {
 		add(type, is64Bit, isARM, null);
 	}
 
-	public void add(TargetOs type, Action<BuildTarget> container) {
-		add(type, false, false, container);
+	public void add(Os type, Action<BuildTarget> container) {
```

Comment:
The following method override is missing for backwards compatibility:
```java
	public void add(Os type, Architecture.Bitness bitness, Action<BuildTarget> container) {
		add(type, bitness, Architecture.x86, container);
	}
```

Reply:
Well noticed!